client/blocks/product-purchase-features-list/test/product-purchase-features-list.jsx

```diff
@@ -63,205 +70,219 @@ describe( 'ProductPurchaseFeaturesList getFeatures() tests', () => {
 	} );
 
 	test( 'should not render features if is placeholder', () => {
-		const comp = shallow( <ProductPurchaseFeaturesList { ...props } isPlaceholder={ true } /> );
-		expect( comp.find( '.product-purchase-features-list' ).children() ).toHaveLength( 0 );
+		const { container } = render(
+			<ProductPurchaseFeaturesList { ...props } isPlaceholder={ true } />
```

Comment:
Now that we're touching this line, we could omit the redundant `={ true }`

```suggestion
			<ProductPurchaseFeaturesList { ...props } isPlaceholder />
```